src/app/page.tsx

```diff
@@ -65,12 +54,12 @@ export default function Home({
   return (
     <main className="min-h-screen flex flex-col">
       <div className="flex justify-between items-center p-4">
-        <h1 className="text-xl font-semibold flex items-center">
+        <h1 className="text-xl font-semibold flex items-center cursor-pointer" onClick={() => window.location.href = '/'}>
           <Image
             src="/metabase.webp"
             alt="Metabase Logo"
             width={150}
-            height={40}
+            height={40}  
```

Comment:
Please include some formatting rules and format on save to avoid extra whitespaces, and to have the consistency across commits. 🙏